ahoopes16/helianthus
# New Logo/Favicon

# Summary
The original favicon I was using was a random picture from the internet. That was fine for a placeholder image, but I wanted to make my own logo for a more personal touch. This PR replaces the old favicon with the new logo that I've created.

This PR closes #10.

If accepted, this PR will:
- Replace all favicon files with files of new logo

# How to Test
1. Pull down this branch and run `yarn && yarn start`
2. Visit http://localhost:3000
3. You should see the following favicon in your browser tab:
![new-favicon](https://user-images.githubusercontent.com/12129503/104831276-9e318d00-5844-11eb-85a6-c6f04a34fedc.png)

You should **no longer** see this one:
![old-favicon](https://user-images.githubusercontent.com/12129503/104831287-a984b880-5844-11eb-835f-c63081ccc23c.png)

# Suggested Reading
- [Favicon Generator](https://realfavicongenerator.net/)
